config/global.ini.php

```diff
@@ -705,6 +705,9 @@
 ; breaks and reencrypts SSL connections you can set your custom file here. 
 custom_cacert_pem=
 
+; set to 1 to copy downloaded plugins to all plugins directories including those defined in MATOMO_PLUGIN_DIRS env variable 
+copy_to_plugins_dirs = 0
```

Comment:
@nabiltntn AFAIK `MATOMO_PLUGIN_DIRS ` is either defined as an environment variable or GLOBALS (through bootstrap.php). Should we have the same for this directory that it can be only configured as a GLOBAL and/or ENV variable? Because the feature should only work if the `copy to plugin directory` is also defined as a `plugin directory`.

Reply:
I changed the implementation to use a new env var : `MATOMO_PLUGIN_COPY_DIR`